SpinKitPouringHourglass name changed

**Describe the bug**
Try correcting the name to the name of an existing method, or defining a method named 'SpinKitPouringHourglass'.
        _indicator = SpinKitPouringHourglass(
                     ^^^^^^^^^^^^^^^^^^^^^^^
                                                                        

FAILURE: Build failed with an exception.

The dependent flutter_spinkit package upgraded and the Class name **SpinKitPouringHourglass** changed to **SpinKitPouringHourGlass** , thus build error occures

The related link is [here](https://github.com/jogboms/flutter_spinkit/releases/tag/v5.1.0)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,7 @@
 ## [3.0.1] - 2021.08.28
 
 * fixed [#120](https://github.com/nslog11/flutter_easyloading/issues/120)
-* 
+
 ## [3.0.0] - 2021.03.12
 
 * 🎉 It's support Null Safety now 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## [3.0.1] - 2021.08.28
+
+* fixed [#120](https://github.com/nslog11/flutter_easyloading/issues/120)
+* 
 ## [3.0.0] - 2021.03.12
 
 * 🎉 It's support Null Safety now 

--- a/README-zh_CN.md
+++ b/README-zh_CN.md
@@ -16,7 +16,7 @@
 
 ```yaml
 dependencies:
-  flutter_easyloading: ^3.0.0
+  flutter_easyloading: ^3.0.1
 ```
 
 ## 导入

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ Add this to your package's `pubspec.yaml` file:
 
 ```yaml
 dependencies:
-  flutter_easyloading: ^3.0.0
+  flutter_easyloading: ^3.0.1
 ```
 
 ## Import

--- a/lib/src/widgets/indicator.dart
+++ b/lib/src/widgets/indicator.dart
@@ -163,7 +163,7 @@ class _LoadingIndicatorState extends State<LoadingIndicator> {
         );
         break;
       case EasyLoadingIndicatorType.pouringHourGlass:
-        _indicator = SpinKitPouringHourglass(
+        _indicator = SpinKitPouringHourGlass(
           color: _indicatorColor,
           size: _size,
         );

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: flutter_easyloading
 description: ✨A clean and lightweight loading/toast widget for Flutter, Easy to use without context, Support iOS、Android and Web
-version: 3.0.0
+version: 3.0.1
 homepage: https://github.com/nslog11/flutter_easyloading
 
 environment:
@@ -10,7 +10,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  flutter_spinkit: ^5.0.0
+  flutter_spinkit: ^5.1.0
 
 dev_dependencies:
   flutter_test:

